multi-agent/tests/unit/resources/test_builtin_conditional_secret_gate.py

```
@@ -0,0 +1,98 @@
+    bearer_token: str | None = Field(
+        default=None,
+        json_schema_extra=combine_hints(
+            SecretHint(),
+            ReadOnlyHint(read_only=False),
+            ConditionalHint(visible_when={"auth_method": "access_token"}),
+        ),
+    )
+
+
+class FakeElementRegistry:
+    def get_schema(self, category, type_key):
```

Ruff (0.16.0)

[warning] Unused method argument: `category`

(ARG002)